src/lib/webpack/transform-config.js

```diff
@@ -1,28 +1,29 @@
-import path from 'path';
-import fs from 'fs.promised';
+import { resolve } from 'path';
 import webpack from 'webpack';
+import fs from 'fs.promised';
+
+const FILE = 'preact.config.js';
 
-export default async function (env, config, ssr = false) {
-	let transformerPath = path.resolve(env.cwd, env.config || './preact.config.js');
+export default async function (env, config, ssr=false) {
+	let myConfig = resolve(env.cwd, env.config);
 
 	try {
-		await fs.stat(transformerPath);
+		await fs.stat(myConfig);
 	} catch (e) {
-		if (env.config) {
-			throw new Error(`preact-cli config could not be loaded!\nFile ${env.config} not found.`);
-		}
-		return;
+		if (env.config === FILE) return;
+		throw new Error(`preact-cli config could not be loaded!\nFile ${env.config} not found.`);
 	}
 
 	require('babel-register')({
```

Comment:
I think we could probably really cut down further on init time if we found a way to avoid pulling in babel-register here.  Maybe `buble-register` as a faster stand-in?

Reply:
Yeah, Buble would certainly be faster, but we're only pulling in Babel in the first place because of 1) async and 2) imports.

I'd advocate for Nodent instead for `async` (I found out how to avoid all the overhead).  And we can use Reify for the imports.

All of this drops any Babel runtime code and I think the new stats would be even better.